website/docs/cookbooks/scala-jvm.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Test your code with java 8 
+sidebar_position: 4
+---
+
+You can use Scala CLI to test your code compatibility with various versions of `java` (no manual installation of java is required).It automatically downloads the specified version of java.
+
+The following snippet used the new method `Files.writeString` from `java` 11. 
+
+```scala name:Main.scala
+import java.nio.file.Files
+import java.nio.file.Paths
+
+object Main extends App {
+    val filePath = Files.writeString(
+      Files.createTempFile(
+        Paths.get("").toAbsolutePath(),
+        "scala-cli",
+        ".txt"
+      ),
+      "Hello from ScalaCli"
+    )
+    val fileContent: String = Files.readString(filePath)
+  }
+ ```
+
+
+Passing `--jvm` to the `scala-cli` command and run your application with the specified java version.
+
+```scala-cli 
+scala-cli Main.scala --jvm adopt:11
```

Comment:
@alexarchambault would it be possible to specify just `--jvm 11` that will default to adopt-11?

Reply:
Yes, it is possible to specify just `--jvm 11` or `--jvm 8` etc. AdoptOpenJDK is used by default.

Here is more details about resolving jvm - [coursier](https://get-coursier.io/docs/cli-java#short-jvm-names)

Reply:
We could default to Java 11. The default JVM id is currently defined [here](https://github.com/VirtusLab/scala-cli/blob/46416b11d3a15ad782dd3f371c02d87e7bb4f749/modules/build/src/main/scala/scala/build/internal/OsLibc.scala#L60-L62).